qiita_ware/util.py

```diff
@@ -117,104 +116,14 @@ def stats_from_df(df):
     out = {}
 
     for column in natsorted(df.columns):
+        if column in {'study_id'}:
+            continue
         counts = df[column].value_counts()
 
         # get a pandas series of the value-count pairs
-        out[column] = [(key, counts[key]) for key in natsorted(counts.index)]
-
-    return out
-
-
-def metadata_stats_from_sample_and_prep_templates(st, pt):
```

Comment:
Was not used in any of the codebase, and does the same thing as `metadata_stats_from_template`